readme.md

```diff
@@ -1,82 +1,92 @@
+
+## Table of Contents
+
+- [Usage](#usage)
+- [Tested Devices](#tested-devices)
+- [Installation](#installation)
+  - [Latest Version (Recommended)](#latest-version-recommended)
+- [Building from Source](#building-application-and-installer-from-source)
+- [Supported Devices](#supported-devices)
+- [Troubleshooting](#troubleshooting)
+- [Uninstallation](#uninstallation)
+- [Acknowledgements](#acknowledgements)
+- [License](#license)
 
 ## Usage
 
-Once started, you can hover over the icon to see the battery level. On right
-click, you can exit the app or see the battery level again.
+Once the application is started, you can hover over the icon to see the battery level. Right-clicking allows you to exit the application or view the battery level again.
 
-## Tested on
+## Tested Devices
 
 - Steelseries AEROX 3 Wireless (2.4G mode)
-- Steelseries Prime Wireless
 - Steelseries AEROX 9 Wireless (2.4G mode)
+- Steelseries Prime Wireless
+
+## Installation
+
+### Latest Version (Recommended)
+
+1. Download the latest application installer from the [Releases](https://github.com/yurtemre7/mouse-battery/releases/) tab.
+2. Run the installer. This will install the application and place a shortcut in your Start Menu and add it to your auto-startup folder.
+3. After installation, the installer will ask you if you want to run the application. If you choose not to, you can run it from the Start Menu shortcut or by restarting your computer.
 
-## Installation 
-0. Download the newest App from the [releases](https://github.com/yurtemre7/mouse-battery/releases/) tab.
-1. Put the `mouse.exe` file in your startup folder 
-   (C:\ProgramData\Microsoft\Windows\Start Menu\Programs\Startup).
-2. Start the program manually or restart your PC.
+#### Manual installation
 
-## Installation (old)
-0. Download the git repository as a zip file and extract it somewhere (or clone
-   it with git)
-1. Install Python 3 (https://www.python.org/downloads/) and make sure to check
-   the box to add it to your PATH.
-2. Pip install the following packages:
-   - `pip install rivalcfg==4.11.0`
-   - `pip install pystray==0.19.5`
-   - `pip install pillow==10.0.0`
-3. Run the script
-   - Put the `start_mouse.bat` file as a shortcut in your startup folder
-     (C:\ProgramData\Microsoft\Windows\Start Menu\Programs\Startup).
-   - You may need to change the bat script to point to your python installation
-     if you have multiple versions installed or renamed the executable.
-   - or you can run the script manually: `python3 mouse.py`
+1. Download the Git repository as a zip file and extract it somewhere (or clone it with Git).
+2. Install Python 3 (<https://www.python.org/downloads/>) and ensure you check the box to add it to your PATH.
+3. Install the following packages with Pip:
+   - `pip install rivalcfg`
+   - `pip install pystray`
+   - `pip install pillow`
```

Comment:
Please reset the old pip installs with the specific versions, as I only tested on them. x)

Reply:
resolved